forge-gui/res/cardsfolder/upcoming/cyclops_superconductor.txt

```diff
@@ -0,0 +1,12 @@
+Name:Cyclops Superconductor
+ManaCost:1 U R
+Types:Creature Cyclops Wizard
+PT:2/2
+K:Prowess
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigEnergy | TriggerDescription$ When CARDNAME enters the battlefield, you get {E}{E}{E} (three energy counters).
+SVar:TrigEnergy:DB$ PutCounter | Defined$ You | CounterType$ ENERGY | CounterNum$ 3
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ DBTrigger | TriggerDescription$ When CARDNAME dies, you may pay {E}{E}{E}. When you do, CARDNAME deals damage equal to its power to any target.
+SVar:DBTrigger:AB$ ImmediateTrigger | Cost$ PayEnergy<3> | Execute$ TrigDamage | TriggerDescription$ When you do, CARDNAME deals damage equal to its power to any target.
+SVar:TrigDamage:DB$ DealDamage | ValidTgts$ Any | NumDmg$ X
+SVar:X:Count$CardPower

```

Comment:
I think for Prowess we usually do just DeckHints:Type$Instant|Sorcery ?
Maybe `BuffedBy:nonCreature` would be good for these too

Reply:
There's a nice little thread about this starting in the Developers discord channel